packages/components/table/src/table.css

```diff
@@ -0,0 +1,140 @@
+/* General........................................................ */
+
+:host {
+	--ts-table-small: 10%;
+	--ts-table-medium: 15%;
+	--ts-table-large: 20%;
+}
+
+table,
+tr {
+	border: 1px solid #d5dfe3;
```

Comment:
Please, use colours from the `packages/core/src/vars.css`, there are colours approved by our designers.
Also, we have predefined styles for borders in the same file, may be you can re-use them here.

Reply:
done